### PR TITLE
[DOCS] Create New Content for Column Descriptive Metrics in GX Cloud 

### PR DESCRIPTION
**Pull request by kwcanuck**
[DSB-364](https://greatexpectations.atlassian.net/browse/DSB-364) requested the addition of a topic to instruct GX Cloud users on how to access and interpret Column Descriptive Metrics (CDM). This PR implements this new content.

[DSB-364]: https://greatexpectations.atlassian.net/browse/DSB-364?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by ghost** [comment]
<details open>
<summary><b>👇 <a href="https://app.codesee.io/r/reviews?pr=8814&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations">Click on the image</a> for a new way to code review</b>
</summary>

####

[Review these changes using an interactive CodeSee Map](https://app.codesee.io/r/reviews?pr=8814&src=https%3A%2F%2Fgithub.com%2Fgreat-expectations%2Fgreat_expectations)

#### Legend
CodeSee Map legend
</details>

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md [hunk 1]
```diff
@@ -67,6 +67,18 @@ Create a Data Asset to define the data you want GX Cloud to access. Currently, t
 
 Data Asset metrics provide you with insight into the data you can use with your Expectations and Validations.
```
Comment (talagluck) [inline docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]:
```suggestion
Data Asset metrics provide you with insight into the data you can use for your data validations. 
```

**Review by talagluck — COMMENTED** [review]
Overall looks good! One small wording update.

File: docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md (hunk 1 above)
Comment (kwcanuck) [reply docs/docusaurus/docs/cloud/data_assets/manage_data_assets.md]: @talagluck your change has been implemented

**Review by talagluck — APPROVED** [review]
LGTM

**Comment by erik-hencier** [comment]
Looks good to me. Tal & Anthony are close to the feature being released as early as today. Thanks Rob!

**Review by anthonyburdi — APPROVED** [review]
Thanks! LGTM!

**Review by r34ctor — APPROVED** [review]
LGTM!

**Pull request merged**